packages/vuetify/src/components/VMenu/VMenu.ts

```diff
@@ -350,6 +350,9 @@ export default baseMixins.extend({
             if (this.closeOnContentClick) this.isActive = false
           },
           keydown: this.onKeyDown,
+          ...this.$listeners.scroll
+            ? { scroll: this.$listeners.scroll }
```

Comment:
Does `undefined` do something bad?

Reply:
Vue throws because of it and there are some `TypeErrors` later along the way.

Reply:
We could `scroll: this.$listeners.scroll || false` (yes, false works :man_shrugging:) or `scroll: this.$listeners.scroll || (() => {})` but it leaves us with listener that does nothing.

Reply:
Alternatively:
```js
      if (this.$listeners.scroll) {
        options.on = options.on || {}
        options.on.scroll = this.$listeners.scroll
      }
```

Reply:
Yeah I like that better, and we're already doing it for the mouse handlers.